### DISCUSSION TITLE
Have tabs along the top bar like shortcuts that only show posts on a particular category

### DISCUSSION THREAD

**Discussion by LiamOrrill** [dailydotdev/daily#627] [Feature Requests]
For example, having categories along the top like Blockchain, Serverless, Frontend, Backend, DevOps etc. Then when you press one it only show's articles related to the topic you have chosen.

**Comment by idoshamun** [comment]
Hi @CryptoLiam,

Thanks for your suggestion. I think it's very much related to this: https://github.com/dailydotdev/daily/discussions/547
What do you think?
